.github/workflows/test.yml

```
@@ -0,0 +1,70 @@
+# things not included
+# git depth (not an issue with gh actions)
+# languaage
+# notifications - no email notifications set up
+# completely lost on stages name/if section
+
+
+name: pytest
+on:
+  pull_request:
+    branches:
+    - '*'
+jobs:
+  default:
+    name: Pytest on ${{ matrix.python-version }}, ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ['ubuntu-latest']
+        python-version: [3.7]
+    timeout-minutes: 30
+    env:
+      DESC: "Python 3.6 tests"  # f strings? is this even necessary?
+      HV_REQUIREMENTS: "unit_tests"
+      PYTHON_VERSION: ${{ matrix.python-version }}
+      PKG_TEST_PYTHON: "--test-python=py37 --test-python=py27"
+      CHANS_DEV: "-c pyviz/label/dev -c bokeh"
+      CHANS: "-c pyviz"
+      MPLBACKEND: "Agg"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: pre_install
+        run: |
+          python -m pip install --upgrade pip
+          pip install pyctdev flake8 nose
+          doit miniconda_install
+      - name: conda setup
+        run: |
+          export PATH="$HOME/miniconda/bin:$PATH"
```

Would try this so the path gets rehashed:

```suggestion
          export PATH="$HOME/miniconda/bin:$PATH" && hash -r
```

---

ah, I took that out at one point to test what was happening. I was concerned that it was placing miniconda and the env in a weird place.